<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Desktop" failures="1" tests="3" name="getFailedScenarioRunner.TestRunner" time="120.954" errors="0" timestamp="2023-02-22T09:54:45 IST" skipped="0">
  <testcase classname="getFailedScenarioRunner.TestRunner" name="runScenario" time="17.073"/>
  <system-out/>
  <testcase classname="getFailedScenarioRunner.TestRunner" name="runScenario" time="18.817"/>
  <system-out/>
  <testcase classname="getFailedScenarioRunner.TestRunner" name="runScenario" time="85.064">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at stepDefinition.VTigerCompaignTest.verify_that_all_the_created_compaings_are_deleted(VTigerCompaignTest.java:113)
at âœ½.Verify that all the created compaings are deleted(file:///E:/Acceleration/23%20August/BDDCucumberFramework/src/test/resources/features/VTigerCompaign.feature:25)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- getFailedScenarioRunner.TestRunner -->
